docs/guides/installation.md

```diff
@@ -24,10 +24,174 @@ Installation requires the following:
+
+**For most users**
+
+If you do *not* have root privileges on the machine you want
+to run the workflow on *or* if you do not have a Linux machine, please [install
+Singularity][singularity-install] separately and in privileged mode, depending
+on your system. You may have to ask an authorized person (e.g., a systems
+administrator) to do that. This will almost certainly be required if you want
+to run the workflow on a high-performance computing (HPC) cluster. 
+
+> **NOTE:**
+> The workflow has been tested with the following Singularity versions:  
+>  * `v2.6.2`
+>  * `v3.5.2`
+
+After installing Singularity, install the remaining dependencies with:
+```bash
+mamba env create -f install/environment.yml
+```
+
+### As root user on Linux
+
+If you have a Linux machine, as well as root privileges, (e.g., if you plan to
+run the workflow on your own computer), you can execute the following command
+to include Singularity in the Conda environment:
+
+```bash
+mamba env update -f install/environment.root.yml
+```
+
+## 5. Activate ZARP environment
+
+Activate the Conda environment with:
+
+```bash
+conda activate zarp
+```
+
+# Extra installation steps (optional)
+
+## 6. Non-essential dependencies installation
+
+Most tests have additional dependencies. If you are planning to run tests, you
+will need to install these by executing the following command _in your active
+Conda environment_:
+
+```bash
+mamba env update -f install/environment.dev.yml
+```
+
+## 6a. Successful installation tests
+
+We have prepared several tests to check the integrity of the workflow and its
+components. These can be found in subdirectories of the `tests/` directory. 
+The most critical of these tests enable you to execute the entire workflow on a 
+set of small example input files. Note that for this and other tests to complete
+successfully, [additional dependencies](#installing-non-essential-dependencies) 
+need to be installed. 
+Execute one of the following commands to run the test workflow 
+on your local machine:
+* Test workflow on local machine with **Singularity**:
+```bash
+bash tests/test_integration_workflow/test.local.sh
+```
+* Test workflow on local machine with **Conda**:
+```bash
+bash tests/test_integration_workflow_with_conda/test.local.sh
+```
+Execute one of the following commands to run the test workflow 
+on a [Slurm][slurm]-managed high-performance computing (HPC) cluster:
+
+* Test workflow with **Singularity**:
+
+```bash
+bash tests/test_integration_workflow/test.slurm.sh
+```
+* Test workflow with **Conda**:
```

Comment:
The formatting of these "headers" for the individual tests is warbled. Why not just use header notation?